lib/storage/metadata/mongoclient/MongoClientInterface.ts

```diff
@@ -407,7 +407,11 @@ class MongoClientInterface {
         const payload = {
             $set: {
                 _id: bucketName,
-                value: newBucketMD,
+                value: {
+                    ...newBucketMD,
+                    quotaMax: new Long(newBucketMD.quotaMax || '0'),
```

Comment:
should have an associated test, to ensure we can indeed properly create buckets with very large quotas

Reply:
> it fails the type validation

what type validation? payload is untyped (for typescript), right?

> Do we still want such test?

from MongoClientInterface standpoint, it is possible to set a quota (i.e. it uses the `newBucketMD.quotaMax`) which is provided: so i think it would be better indeed to add one (or more: no quota/small quota/large bigint) test case in MongoClientInterface tests

Reply:
> what type validation? payload is untyped (for typescript), right?

We enforce types in the mongodb driver, o we cannot actually create a bucket MD if the quota is not a Long (which is expected). I'll add a test